app/bundles/CampaignBundle/Config/config.php

```diff
@@ -174,6 +174,9 @@
             ],
             'mautic.campaignrange.type.action' => [
                 'class' => 'Mautic\CampaignBundle\Form\Type\EventType',
+                /** CAPTIVEA.CORE START **/
+                'arguments' => 'mautic.factory',
```

Comment:
I didn't saw that it's deprecated, my bad (in which branch was it published ?). What would be an efficient replacement, knowing that:
* we need to access entity manager
* forms are not container aware, so we can't call a service that is not injected
?

Of course, I'll change all other references to factory to use the appropriate service instead.